.env.example

```
@@ -3,28 +3,32 @@ PORT=3000
 APPWRITE_DATABASE_ID=xreporter
 
 # ----- Redis (BullMQ) -----
+# Must use redis:// or rediss:// scheme. `docker compose up -d` exposes 6379.
 REDIS_URL=redis://localhost:6379
 
 # ----- X OAuth2 -----
+# Create an app at https://developer.x.com and enable OAuth2 with PKCE.
 X_CLIENT_ID=
 X_CLIENT_SECRET=
 X_REDIRECT_URI=http://localhost:3000/auth/x/callback
 X_SCOPES=tweet.read users.read like.read bookmark.read offline.access
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)